src/components/Dialog/SettingDialog/SettingDialog.vue

```diff
@@ -961,15 +875,24 @@ const setMultiTrack = (enableMultiTrack: boolean) => {
 <style scoped lang="scss">
 @use "@/styles/visually-hidden" as visually-hidden;
 @use "@/styles/colors" as colors;
+@use "@/styles/v2/colors" as colors-v2;
+@use "@/styles/v2/variables" as vars;
+@use "@/styles/v2/mixin" as mixin;
+
+.container {
+  position: absolute;
+  left: 0;
+  right: 0;
+  height: 100%;
+  background-color: colors-v2.$background;
+}
 
-.text-h5 {
-  margin: 0;
+.headline {
+  @include mixin.headline-2;
 }
 
-.setting-dialog {
-  .q-field__control {
-    color: colors.$primary;
-  }
+:global(.setting-dialog) {
```

Comment:
これ何で`:global`がいるのかわかんないですね。。。。。。。。
なんでいるのかわからないということがわかるようにしとくのはどうでしょう。。。
```suggestion
// NOTE: なぜか:globalしないと効かない。QDialogの仕様かも。
:global(.setting-dialog) {
```

Reply:
実装時もとりあえず試行錯誤してみてそれでいけた感じなので確かにそうですね、、とりあえずコメント付与しました…！